Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/SubmissionResults.tsx

```diff
@@ -42,24 +44,33 @@ const SubmissionResults = ({ problemId }: ISubmissionResultsProps) => {
         }
     }, [ getSubmissionResults, problemId ]);
 
-    useEffect(() => {
+    const reload = () => {
         (async () => {
             await getResults();
         })();
+    };
+
+    useEffect(() => {
+        if (currentProblemSubmissionResults.length !== 0) { return; }
+        reload();
     }, []);
 
     return (
         currentProblemSubmissionResults.length === 0
             ? <p> No results for this problem yet.</p>
             : (
-                <div style={{ height: 300, width: '100%' }}>
-                    <DataGrid
-                      rows={currentProblemSubmissionResults}
-                      columns={columns}
-                      pageSize={5}
-                      rowsPerPageOptions={[ 5 ]}
-                      disableSelectionOnClick
-                    />
+                <div className={styles.submissionResultsContent}>
+                    <div style={{ height: 300, width: '100%' }}>
```

Comment:
This is the reason, I do not like `material`

Reply:
I'm open to a discussion whether we should use it or not.